escher/src/text_input.cpp

```diff
@@ -99,3 +99,22 @@ bool TextInput::removeEndOfLine() {
 bool TextInput::privateRemoveEndOfLine() {
   return contentView()->removeEndOfLine();
 }
+
+bool TextInput::moveCursorLeftRight(bool right) {
```

Comment:
Why would you factorize moveCursorLeft and moveCursorRight if their implementation is a total disjunction?

Reply:
That's right too ! It was not a disjunction when I started cleaning.